DataTests/BookmarkTests.swift

```diff
@@ -274,6 +274,34 @@ class BookmarkTests: CoreDataTestCase {
         XCTAssertEqual(Bookmark.getAllBookmarks().count, bookmarksCount)
     }
     
+    func testHasFavorites() {
+        XCTAssertFalse(Bookmark.hasFavorites)
+        
+        insertBookmarks(amount: 3)
+        
+        XCTAssertFalse(Bookmark.hasFavorites)
+        
+        createAndWait(url: URL(string: "http://brave.com"), title: "Brave", isFavorite: true)
```

Comment:
I realize this is a test, but it looks wrong to see a Brave URL with `http` instead of `https` :)

Reply:
haha definitely, will change :)